src/features/pages/home/index.tsx

```diff
@@ -1,20 +1,20 @@
 /* eslint-disable @typescript-eslint/ban-ts-comment */
-import React from 'react'
+import React, { lazy, Suspense } from 'react'
 import { PageLayout } from '@deriv-com/components'
 import QuillLayout from 'features/components/quill/quill-layout'
-import LiveMarketSection from './live-pricing-migration'
 import TwentyYearsStrong from './years'
 import StatSection from './stats'
 import UserFriendlyPlatforms from './user-platfroms'
-import TradeTypeSection from './trade-type'
 import HomeHero from './hero'
-import FastPaymentSection from './fast-payment'
 import CTA from './cta'
 import StartTradingSteps from './start-trading-steps'
 import TrustpilotSection from './trustpilot'
 import { useOpenLiveChat } from 'components/hooks/use-open-live-chat-redirection'
 import MainRowNavigation from 'features/components/templates/navigation/main-nav'
 import MainFooter from 'features/components/templates/footer'
+const FastPaymentSection = lazy(() => import('./fast-payment'))
```

Comment:
how do we know that this library was actually excluded from main bundle? 
(reason I'm asking is that I used to implement similiar patterns in different project and commonly I have been stumbling upon the fact that some random tiny component was still importing something from file intended to be lazy loaded and thus preventing webpack from removing it from main bundle)

Reply:
it was not, because we compile ALL code into a single file. we do not have chunks as they create too much overhead for the network and almost no win with size.

Reply:
so if its not excluded from main bundle, then whats the point of lazy loading?

Reply:
IMO lazy loading does make sense only if given resource is not already included in my bundle